rpcserver.go

```diff
@@ -148,9 +186,19 @@ func (s *rpcServer) Start() error {
 		return fmt.Errorf("error in GetInfo: %v", err)
 	}
 
-	_, s.wumboSupported = info.Features[uint32(lnwire.WumboChannelsRequired)]
+	_, s.wumboSupported = info.Features[wumboRequired]
 	if !s.wumboSupported {
-		_, s.wumboSupported = info.Features[uint32(lnwire.WumboChannelsOptional)]
+		_, s.wumboSupported = info.Features[wumboOptional]
+	}
+	_, s.taprootChansSupported = info.Features[taprootChansStagingRequired]
```

Comment:
I think we just want something like: 
`s.taprootChansSupported = info.Features[taprootChansStagingRequired] != nil || info.Features[taprootChansStagingOptional] != nil`

etc